src/services/CertificateGenerationService.ts

```diff
@@ -605,7 +605,7 @@ class CertificateGenerationService {
           type: testResult.vehicleType,
           testerName: testResult.testerName,
           date: moment(testResult.testTypes.testTypeStartTimestamp).format("DD/MM/YYYY"),
-          retestDate: this.calculateVehicleApprovalRetestDate(testResult.testTypes.testTypeStartTimestamp),
+          reapplicationDate: testResult.reapplicationDate ?? "",
```

Comment:
does this not need a date format like above? is it not coming as a UTC timestamp?